### PR TITLE
[Security] Added missing Polish translations

### PR DESCRIPTION
**Pull request by ShiroRyokoshi**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix ##41061
| License       | MIT
| Doc PR        | N/A

The translations added were based off the english ones.
This is my first PR ever, I hope I didn't do any mistakes.

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.pl.xlf [hunk 1]
```diff
@@ -69,6 +69,14 @@
...
                 <source>Invalid or expired login link.</source>
                 <target>Nieprawidłowy lub wygasły link logowania.</target>
+			</trans-unit>
+			<trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Zbyt wiele nieudanych prób logowania, spróbuj ponownie po upływie %minutes% minut.</target>
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Zbyt wiele nieudanych prób logowania, spróbuj ponownie po upływie %minutes% minut.</target>
```
Comment (skmedix) [inline src/Symfony/Component/Security/Core/Resources/translations/security.pl.xlf]: Please fix formatting.

```suggestion
            </trans-unit>
            <trans-unit id="19">
                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
                <target>Zbyt wiele nieudanych prób logowania, spróbuj ponownie po upływie %minutes% minut.</target>
            </trans-unit>
            <trans-unit id="20">
                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
                <target>Zbyt wiele nieudanych prób logowania, spróbuj ponownie po upływie %minutes% minut.</target>
```

**Review by skmedix — CHANGES_REQUESTED** [review]

**Review by Nyholm — APPROVED** [review]
Thank you. It looks great!

Could @pamil give this a quick review too?

**Comment by ShiroRyokoshi** [comment]
@Nyholm Hey, can I close this request or are we still waiting for some extra review?

**Comment by fabpot** [comment]
Thank you @ShiroRyokoshi.

**Pull request merged**